modules/core/src/matrix_expressions.cpp

```diff
@@ -420,7 +420,7 @@ void MatOp::multiply(const MatExpr& e1, const MatExpr& e2, MatExpr& res, double
             {
                 op = '/';
                 m2 = e2.a;
-                scale /= e2.alpha;
+                scale = e2.alpha / scale;
```

Comment:
Shouldn't it be `scale *= e2.alpha;`
Otherwise the change will fail following evaluation: `(Mat)(mat.mul((a / mat), 2.))`

Reply:
Thank you for the comment.
It certainly fails in the case :(Mat) (mat.mul ((a / mat), 2.)).
I will correct it soon.